package.json

```diff
@@ -12,10 +12,13 @@
     "postinstall": "cd packages/react-instantsearch && npm install",
     "doctoc": "doctoc --maxlevel 1 README.md CONTRIBUTING.md",
     "test": "(cd packages/react-instantsearch && npm test) && npm run lint && NODE_ENV=production DOCS_MOUNT_POINT=/instantsearch.js/react/ DOCS_DIST=docs/react/ npm run docs:build && NODE_ENV=production npm run build",
-    "dev": "cd packages/react-instantsearch && npm run dev"
+    "dev": "cd packages/react-instantsearch && npm run dev",
+    "storybook": "start-storybook -p 6006",
```

Comment:
Yes you can, and since this week, you are able to do it without a lot of configuration. 
It's their own cli though: https://voice.kadira.io/snapshot-testing-in-react-storybook-43b3b71cec4f#.jjp5n6ivb